src/modules/sentry/sentry.service.ts

```diff
@@ -37,6 +37,20 @@ export class SentryService {
         }),
         new Sentry.Integrations.OnUnhandledRejection({ mode: 'warn' }),
       ],
+      beforeSend(event) {
+        const request = event.request;
+        if (request) {
+          delete request.cookies;
+          delete request.data.identityToken;
```

Comment:
Maybe safe index would be better?
```suggestion
          delete request.data?.identityToken;
```